service/src/main/resources/application.yaml

```diff
@@ -238,10 +238,10 @@ sbomer:
     generation-scheduler:
       # Defines how many requests can be handled concurrently within the namespace
       # If the capacity is lower than what we have in the DB -- the ones in DB will wait
-      max-concurrent-generations: 20
+      max-concurrent-generations: 15
 
       # Maximum number of generations that will fetched from the database to be scheduled within the namespace.
-      sync-batch: 10
+      sync-batch: 5
```

Comment:
I think this will definitely help, If we only had 10 connections before and each generation was possibly doing transactional calls (I think quarkus may hold the connection open for this)